CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 set(AIE_TOOLS_BINARY_DIR ${AIE_BINARY_DIR}/bin)
 
-find_package(Vitis 2022.2 COMPONENTS AIE AIE2)
+find_package(Vitis 2023.2 COMPONENTS AIE AIE2)
```

Comment:
We call `find_package(Vitis 2022.2` a few other places. In reference_designs, test, and tutorials.

Reply:
Good opportunity to make this symbolic?

Reply:
I don't think it's easily symbolic, since these directories are essentially independent cmake projects within the same directory.